ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly/V100_0__Append_only_schema.sql

```diff
@@ -0,0 +1,484 @@
+       NULL          AS create_agreement_text,
+       participant_events_consuming_exercise.create_key_value,
+       NULL          AS create_key_hash,
+       participant_events_consuming_exercise.exercise_choice,
+       participant_events_consuming_exercise.exercise_argument,
+       participant_events_consuming_exercise.exercise_result,
+       participant_events_consuming_exercise.exercise_actors,
+       participant_events_consuming_exercise.exercise_child_event_ids,
+       NULL          AS create_argument_compression,
+       participant_events_consuming_exercise.create_key_value_compression,
+       participant_events_consuming_exercise.exercise_argument_compression,
+       participant_events_consuming_exercise.exercise_result_compression
+FROM participant_events_consuming_exercise
+UNION ALL
+SELECT (25)          AS event_kind,
+       participant_events_non_consuming_exercise.event_sequential_id,
+       participant_events_non_consuming_exercise.event_offset,
+       participant_events_non_consuming_exercise.transaction_id,
+       participant_events_non_consuming_exercise.ledger_effective_time,
+       participant_events_non_consuming_exercise.command_id,
+       participant_events_non_consuming_exercise.workflow_id,
+       participant_events_non_consuming_exercise.application_id,
+       participant_events_non_consuming_exercise.submitters,
+       participant_events_non_consuming_exercise.node_index,
+       participant_events_non_consuming_exercise.event_id,
+       participant_events_non_consuming_exercise.contract_id,
+       participant_events_non_consuming_exercise.template_id,
+       participant_events_non_consuming_exercise.flat_event_witnesses,
+       participant_events_non_consuming_exercise.tree_event_witnesses,
+       EMPTY_BLOB()  AS create_argument,
+       to_clob('[]') AS create_signatories,
+       to_clob('[]') AS create_observers,
+       NULL          AS create_agreement_text,
+       participant_events_non_consuming_exercise.create_key_value,
+       NULL          AS create_key_hash,
+       participant_events_non_consuming_exercise.exercise_choice,
+       participant_events_non_consuming_exercise.exercise_argument,
+       participant_events_non_consuming_exercise.exercise_result,
+       participant_events_non_consuming_exercise.exercise_actors,
+       participant_events_non_consuming_exercise.exercise_child_event_ids,
+       NULL          AS create_argument_compression,
+       participant_events_non_consuming_exercise.create_key_value_compression,
+       participant_events_non_consuming_exercise.exercise_argument_compression,
+       participant_events_non_consuming_exercise.exercise_result_compression
+FROM participant_events_non_consuming_exercise;
+
+
+-- Stores which events were touched by which migration.
+-- This metadata is not used for normal indexing operations and exists only to simplify fixing data migration issues.
+CREATE TABLE participant_migration_history_v100 (
```

Comment:
you do not need that, because no datacontinuity is needed for append only

Reply:
i do want this for oracle to avoid repetition of the initial definitions and have the old and new schema working for comparison -- so this is in the same style as postgres with postgres and postgres-appendonly

Reply:
removed